rancher-cluster-operations/.terraform.lock.hcl

```diff
@@ -5,6 +5,7 @@ provider "registry.terraform.io/hashicorp/kubernetes" {
   version = "2.6.1"
   hashes = [
     "h1:DWgawNO2C7IuXC2v9IjTSsqs1vZHSAbP4ilWQ0LdbwI=",
+    "h1:xC0zk0wWTlWLwGLp9cBRJG8672Pp1PZQmHCA4FAJp4U=",
```

Comment:
Nit: This PR should only include control-plane module updates

Reply:
Same. Not sure how this got in there, since I just merged upstream into my fork. I didn't even do a `terraform init`